### ISSUE TITLE
"Show alt text" incorrectly flags missing alt text

### ISSUE THREAD

**Issue by swashbuck** [cgkineo/adapt-devtools#106]
### Subject of the issue
When using the "show alt text" option in Dev Tools, images with alt text will sometimes display "No alt text". This has been flagged by clients while reviewing a course for accessibility compliance.

https://github.com/cgkineo/adapt-devtools/blob/289857f758617738880a5ef4900da78b6438b2f7/templates/devtoolsAnnotation.hbs#L5-L9

### Your environment
* Dev Tools 3.7.1

### Steps to reproduce
As an example, in _page.hbs_, add the following HTML after the opening `.page__header-inner` tag:

```
<div class="page__image-container">
  <img class="page__image" src="course/en/images/logo.png" alt="Testing logo" style="width: 120px; min-width: 0;">
</div>
```

Note that the logo shows "no alt text" despite having an alt property.

### Screenshots
![Image](https://github.com/user-attachments/assets/7dbbb2c8-16da-4674-bb0d-f98f0a57c8cd)
